subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/pmd/AbstractPmdPluginVersionIntegrationTest.groovy

```diff
@@ -22,7 +22,7 @@ import org.gradle.integtests.fixtures.TargetVersions
 import org.gradle.util.TestPrecondition
 import org.gradle.util.VersionNumber
 
-@TargetVersions([PmdPlugin.DEFAULT_PMD_VERSION, '4.3', '5.0.5', '5.1.1', '5.3.3', '6.0.0', '6.20.0'])
+@TargetVersions([PmdPlugin.DEFAULT_PMD_VERSION, '4.3', '5.0.5', '5.1.1', '5.3.3', '6.0.0', '6.20.0', '6.21.0'])
```

Comment:
Since you're making the default version 6.21.0, we don't need to test it separately here. 6.20.0 probably shouldn't have been here either and we should have tested an earlier version (between 6.0 and 6.20).

I would just leave 6.20.0 here and not add 6.21.0.

Reply:
Sorry, I completely overlooked at that we add the default version already. Removed new default version and replaced the previous one with an older version. I scrolled through the release notes and with 6.8.0, they started with a more strict API/internals model so I thought that might be a good candidate to have in our tests.